Removed header files no longer include in recent versions of protobuf.
--- plugin/x/ngs/include/ngs/protocol/protocol_protobuf.h.orig	2019-08-14 07:25:47.000000000 -0700
+++ plugin/x/ngs/include/ngs/protocol/protocol_protobuf.h	2019-09-17 20:10:21.000000000 -0700
@@ -38,7 +38,6 @@
 #include <google/protobuf/repeated_field.h>
 #include <google/protobuf/text_format.h>
 #include <google/protobuf/wire_format_lite.h>
-#include <google/protobuf/wire_format_lite_inl.h>
 
 #ifdef USE_MYSQLX_FULL_PROTO
 #include "plugin/x/generated/protobuf/mysqlx.pb.h"
--- plugin/x/client/mysqlxclient/xmessage.h.orig	2019-08-14 07:25:47.000000000 -0700
+++ plugin/x/client/mysqlxclient/xmessage.h	2019-09-17 20:10:41.000000000 -0700
@@ -36,7 +36,6 @@
 #include <google/protobuf/repeated_field.h>
 #include <google/protobuf/text_format.h>
 #include <google/protobuf/wire_format_lite.h>
-#include <google/protobuf/wire_format_lite_inl.h>
 
 #ifdef USE_MYSQLX_FULL_PROTO
 
